### DISCUSSION TITLE
I have Error: account/login-panel/server.lua:372: attempt to call global 'bcrypt_gensalt' (a nil value)

### DISCUSSION THREAD

**Discussion by Majorrek** [OwlGamingCommunity/MTA#8] [Support] [answered]
Hi i got this error on 32 bit and 64 bit linux server? Does anyone have a solution?

**Comment by GilroyMTA** [comment]
Same here. When you fix 372, 96 pops out as an error.

**Comment by Majorrek** [comment]
> 
> 
> Same here. When you fix 372, 96 pops out as an error.

Yes, did you find any solution?

**Comment by GilroyMTA** [comment]
No. Just waiting for the answer here.

**Comment by ChaosParadise** [comment]
It means your bcrypt module hasn't loaded properly. MTA should say early in your logs as to why.

**Reply by GilroyMTA** [reply]
No reason is given in the server.logs

`MODULE: Unable to load mods\deathmatch\modules\bcrypt.dll (126)`

**Comment by saviorsoldier** [comment, marked as answer]
I'm using Win Server 2012. Downloading latest Visual C++ Redistributable solved the issue. So you guys need to check dependencies of the module.

**Reply by GilroyMTA** [reply]
Microsoft Visual C++ 2015 Redistributable fixes but now it is blocking the texture-system. We are losing the connection when we are setting this up. Like in I mentioned it here: https://github.com/OwlGamingCommunity/MTA/discussions/6
